src/main/java/com/codejoust/main/dto/game/PlayerDto.java

```diff
@@ -19,6 +19,6 @@
     private String code;
     private CodeLanguage language;
     private List<SubmissionDto> submissions = new ArrayList<>();
-    private Boolean[] solved;
+    private boolean[] solved;
```

Comment:
What was your thinking for changing these all to primitives? I'm not really sure of the pros / cons here of using primitives vs. objects, thoughts @jasonz6688 and @alankbi?

Reply:
Not sure either - I guess it simplifies things by making them default false rather than null? With single boolean values ik it changes the getter name, but idk if that's relevant here when it's an array

Reply:
Yeah, the primary reason was that initialization now makes an array of falses because otherwise, we would have to iterate through the array and put initialize a false boolean object.